modules/ztp-pre-hub-install-factory-pipeline-checks.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: PROCEDURE
+[id="verify-the-hub-cluster-is-ready_{context}"]
+= Verify the hub cluster is ready to run the factory install pipeline
+
+Run the following steps to ensure the the hub cluster is ready to run the factory install pipeline.
+
+.Prerequisites
+
+* An {product-title} hub cluster
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Verify the status of the nodes:
++
+[source,terminal]
+----
+$ oc get nodes
+----
++
+.Example output
++
+[source,terminal]
```

Comment:
Labels need to be on the line immediately before blocks that they describe. Apply across PR. 

```suggestion
.Example output
[source,terminal]
```